demo/src/screens/componentScreens/OverlaysScreen.js

```diff
@@ -0,0 +1,93 @@
+import React, {Component} from 'react';
+import {StyleSheet, ScrollView} from 'react-native';
+import {View, Text, Image} from 'react-native-ui-lib'; // eslint-disable-line
+
+
+const image = require('../../assets/images/card-example.jpg');
```

Comment:
I think it will better to show the overlay on big assets like the placeholder illustrations from the private repo (one is enough)

Reply:
Not sure I understand what you mean...